packages/react-core/src/components/Modal/Modal.tsx

```diff
@@ -41,6 +41,10 @@ export interface ModalProps extends React.HTMLProps<HTMLDivElement>, OUIAProps {
   description?: React.ReactNode;
   /** Variant of the modal */
   variant?: 'small' | 'medium' | 'large' | 'default';
+  /** Alternate position of the modal */
+  position?: 'top';
+  /** Custom distance from top */
+  distanceFromTop?: string;
```

Comment:
if we support a generic `position` prop, can we have a generic `distance` prop?

Reply:
Sure, I'll update this!